packages/react-composites/.eslintrc.js

```diff
@@ -105,6 +107,12 @@ module.exports = {
       rules: {
         'jsdoc/require-jsdoc': 'off'
       }
+    },
+    {
+      files: ['*.test.ts', 'AzureCommunicationCallAdapter.ts'],
```

Comment:
Does the chat adapter have to be included here?

Reply:
It didn't have any errors on it, but you're right it definitely could and it doesn't need gated, I'll update that in this PR https://github.com/Azure/communication-ui-library/pull/5239

Reply:
Updated